frappe/email/doctype/email_account/email_account.js

```diff
@@ -119,7 +119,8 @@ frappe.ui.form.on("Email Account", {
 	},
 
 	show_gmail_message_for_less_secure_apps: function(frm) {
-		if(frm.doc.service==="Gmail") {
+		if(frm.doc.service==="GMail") {
+			frm.dashboard.clear_headline();
```

Comment:
Should be before the `if` condition. Otherwise headline is not cleared on Service change.